dre-service/src/main/java/io/cdap/re/RulesDB.java

```diff
@@ -525,13 +600,21 @@ private long getCurrentTime() {
 
   private List<String> convertRulesToSet(String rules) {
```

Comment:
this method should not exist, use Gson to deserialize instead of doing manual string parsing.

Reply:
Done